cmake/Platform/Linux/PackagingPostBuild_linux.cmake

```diff
@@ -56,7 +58,7 @@ if(CPACK_UPLOAD_URL)
     ly_upload_to_url(
         ${CPACK_UPLOAD_URL}
         ${CPACK_UPLOAD_DIRECTORY}
-        ".*(.deb|.gpg|.sha256|.snap|.assert|.txt|.json)$"
+        ".*(.deb|.gpg|.sha256|.sha384|.snap|.assert|.txt|.json)$"
```

Comment:
do we need the `.assert` in here any more?

Reply:
Not anymore. I'll remove it